Cargo.lock

```diff
@@ -6491,7 +6500,7 @@ dependencies = [
 [[package]]
 name = "selectors"
 version = "0.26.0"
-source = "git+https://github.com/servo/stylo?branch=2025-03-01#0190fff684fa7b67f5413a60f215e671b6222534"
+source = "git+https://github.com/servo/stylo?rev=refs%2Fpull%2F102%2Fhead#c297870b9b536c576ec24c6a21067bf19ec80685"
```

Comment:
You need to update this PR so that all of these point to your current Stylo commit, https://github.com/servo/stylo/commit/8ec2526808548b50f4c697e702f5e9b5994f8091